nss-day-cohort-60/national-parks-api
# Client can now get photos by user_id AND park_id

# Description
Created a new function **get_photos_by_user_id_and_park_id()** in photo_views.py that allows the client to filter all photos by both a user id and park id.

# Steps to Test 

1. Run the below commands in terminal
```
git fetch origin vs-park-list-photos
cd vs-park-list-photos
```
2. Restart the debugger
3. Open Postman and run the following GET request:
http://localhost:8088/photos?user_id=11&park_id=4
4.  Verify that the API returned a 200 status code and only dictionaries with user_id = 11 and park_id = 4
